articles/implementing-shooting-method-in-matlab/index.md

```diff
@@ -0,0 +1,191 @@
+### Shooting method using Matlab
+### Introduction
+
```

Comment:
You have double headings? I cant tell what the first one does.

Reply:
I just see a single heading, I don't get what you mean